src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -12,12 +12,17 @@
 
 @if (_trace is { } trace)
 {
-    <div class="trace-detail-layout">
-        <div class="trace-detail-header page-header">
-            <h1 style="display:inline-block">@GetResourceName(trace.FirstSpan.Source): @trace.FirstSpan.Name</h1>
-            <span class="trace-id">@OtlpHelpers.ToShortenedId(trace.TraceId)</span>
-        </div>
-        <FluentToolbar Orientation="Orientation.Horizontal">
+    <AspirePageContentLayout
```

Comment:
The open span details is lost when switching views. When this is fixed, make sure the state of the span details stays the same when switching between views.

Reply:
This has been fixed